go.mod

```diff
@@ -220,8 +220,11 @@ require (
 )
 
 replace (
-	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.18.3-sdk-v0.46.14
+	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.20.1-sdk-v0.46.16
+	// Pin to ledger-cosmos-go v0.12.4 to avoid a breaking change introduced in v0.13.0
+	// The following replace statement can be removed when we upgrade to cosmos-sdk >= v0.50.0
+	github.com/cosmos/ledger-cosmos-go => github.com/cosmos/ledger-cosmos-go v0.12.4
```

Comment:
I got this change added as part of merging the PR from main. Do we want this @rootulp?

Reply:
hmm I think yes. IIRC the v0.13.0 breaking change manifests as a Go compile time error so we could hypothetically remove this replace and see if a transitive dep _doesn't_ pull in a v0.13.x release but I think it's safer to pin to the version that Go compiles.